spec/lib/pending_assigns_reminder_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe PendingAssignsReminder do
+  let(:user) { Fabricate(:user) }
+
+  it 'Do not create a reminder if the user has 0 assigned topics' do
```

Comment:
This is very minor but I think we should maintain the same convention that the rest of our specs are using. There are two things here I think we should change: 

1. Our existing tests do not capitalize the first letter of the description
1. it 'does not ...' instead of it 'do not...'